models/message.model.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const messageSchema = new mongoose.Schema(
+  {
+    content: {
+      type: String,
+    },
+    attachment: {
+      type: String,
+    },
+    memberId: [
```

Comment:
again why ref is User , and why its an array a message will be sent by a single member so its unique